ocs_ci/deployment/cnv.py

```diff
@@ -792,3 +797,92 @@ def uninstall_cnv(self, check_cnv_installed=True):
 
         logger.info("Removing the openshift virtualization CRDs")
         self.remove_crds()
+
+    def cnv_patch_subscription(self, patch):
+        patch_cmd = (
+            f"oc -n {self.namespace} patch sub {constants.KUBEVIRT_HYPERCONVERGED} "
+            f"-p {patch} --type merge"
+        )
+        run_cmd(patch_cmd)
+
+    def get_running_cnv_version(self):
+        """
+        Get the currently deployed cnv version
+
+        Returns:
+            string: cnv version
+
+        """
+        occmd = (
+            f"oc get sub {constants.KUBEVIRT_HYPERCONVERGED} -n openshift-cnv -o json"
+        )
+        jq_cmd = "jq -r .status.currentCSV"
```

Comment:
```
hyperconverged_subs = OCP(kind=constants.SUBSCRIPTION_WITH_ACM, namespace=self.namespace, resource_name=constants.KUBEVIRT_HYPERCONVERGED)
cnv_version = hyperconverged_subs.get()["status"]["installedCSV"]
return cnv_version
```

The value of cnv_version will be like "kubevirt-hyperconverged-operator.4.18.1-10"

Reply:
Addressed